src/App/pages/RepoPage/RepoPage.tsx

```diff
@@ -1,73 +1,61 @@
-import React, { useEffect, useState } from "react";
+import React, { useEffect } from "react";
 
 import Avatar from "@components/Avatar";
 import { getUpdateDate } from "@components/RepoTile/RepoTile";
 import StarIcon from "@components/StarIcon/StarIcon";
-import { ApiResponse } from "@shared/store/ApiStore/types";
-import GitHubStore from "@store/GitHubStore";
-import { RepoItem } from "@store/GitHubStore/types";
-import { useParams, Navigate } from "react-router-dom";
+import RepoItemStore from "@store/RepoItemStore";
+import { Meta } from "@utils/meta";
+import { useLocalStore } from "@utils/UseLocalStore";
+import { observer } from "mobx-react-lite";
+import { Navigate, useParams } from "react-router-dom";
 
 import styles from "./RepoPage.module.scss";
-const gitHubStore = new GitHubStore();
-const RepoPage = () => {
+
+const RepoPage: React.FC = () => {
   const { id } = useParams();
-  const [repo, setRepo] = useState<null | RepoItem>(null);
-  const [error, setError] = React.useState(null);
+  const RepoStore = useLocalStore(() => new RepoItemStore());

```

Comment:
Если это значение, то с маленькой буквы, в конструктор стора сразу можно прокинуть id, это логично, так как стор отвечает за конкретный репозиторий
```suggestion
  const repoStore = useLocalStore(() => new RepoItemStore(id));

```